qa/L0_onnx_optimization/test.sh

```diff
@@ -60,6 +60,20 @@ for MODEL in \
     rm -fr models/${MODEL}_test/3 && \
     (cd models/${MODEL}_test && \
             sed -i 's/_float32_float32_float32/&_test/' config.pbtxt) && \
+    # CUDA EP optimization params
+    cp -r models/${MODEL}_test models/${MODEL}_cuda_config && \
```

Comment:
You've changed the directory name from L0_onnxtrt_optimization to L0_onnx_optimization... is that intentional? If so I need to adjust on CI side as well.

Reply:
Yes it is intentional. I made this change because now I added tests for CUDA provider as well... so this is not more a onnx-trt dedicated test.